mantidproject/mantid
# Apply clang-tidy modernize-use-override to testing headers

Description of work.

This uses clang-tidy's modernize-use-override tool to add the C++11 override keyword where appropriate. This should eliminate many of the warnings emitted by GCC 5's -Wsuggest-override flag.

Unlike PR #15308, this applies changes to the testing headers.

**To test:**



This is a relatively simple keyword change done with automated tooling. Rather than reviewing every change I suggest reviewing enough files to convince yourself clang-tidy is doing the right things. A misspelled or duplicated keyword should not build.

This PR does not have an associated issue.

Does not need to be in the release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
